adam/experiment/m6_experiments.py

```diff
@@ -0,0 +1,41 @@
+from vistautils.parameters import Parameters
+from vistautils.parameters_only_entrypoint import parameters_only_entry_point
+
+from adam.curriculum.m6_curriculum import make_m6_curriculum
+from adam.experiment import execute_experiment, Experiment
+from adam.experiment.observer import HTMLLogger, HTMLLoggerObserver
+from adam.learner.subset import SubsetLanguageLearner
+from adam.random_utils import RandomChooser
+
+
+def main(params: Parameters) -> None:
+    output_dir = params.creatable_directory("output_directory")
+    experiment_name = params.optional_string("experiment", default="m6_experiments")
+
+    logger = HTMLLogger(
+        output_dir=output_dir,
+        experiment_name=experiment_name,
+        curriculum_name="m6_curriculum",
+    )
+
+    m6_curriculum = make_m6_curriculum()
```

Comment:
nit: you might as well in-line this below, since nothing else is done with it and the name doesn't clarify anything